### ISSUE TITLE
[NEW ITEM] Do Not Touch This Village

### ISSUE THREAD

**Issue by KadTheHunter** [KadTheHunter/ShulkerArchives#62]
### Item Name
- Do Not Touch This Village

### Author
- Caketaco/Feueristic

### Description
- A book chronicling the events of the Do Not Touch This Village event

### Item
(e.g diamond_sword)
- writable_book

### NBT Data 
(Paste on the line between backticks)
```
{Enchantments:[{id:"minecraft:fortune",lvl:1s}],HideFlags:1,display:{Lore:['{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Written by Caketaco"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Event by Feueristic"}],"text":""}','{"italic":false,"color":"white","extra":[{"text":""},{"color":"gray","text":"Original"}],"text":""}'],Name:'{"italic":false,"extra":[{"text":""},{"color":"white","text":"Please do not touch this village."}],"text":""}'},pages:["

   §l Please do not
 touch this village§r







      §oby Caketaco§r

   §oEvent by Feueristic§§","


§oA recount of the events leading up to the now-touched village you see before you today.§r","       §o§lBeginnings§r 
       

    It all started with a forum post. On a quiet Thursday night, Feueristic wandered the overworld, found a village, and hatched a brilliant idea.

    They created a warp, took a screen-
shot, and uploaded it ",'


to the forums. The idea was simple. Bold. Unoriginal, yet new.

They posted the details of the village onto the forum with the title:

§o"[Event] Please do not touch this village!§§"§§§§ ',"§7[Event] Please do not touch this village!§r
§8⛄§r§3Feueristic§r§r §8⌚23/1/26§r

Hi!
I hope this forum post find you well.
Today I am here to talk about this village.
I love this village.

§8§o[Image of the village before being touched]§§","§7[Event] Please do not touch this village!§r
§8⛄§r§3Feueristic§r§r §8⌚23/1/26§r

This village is in
§n/warp please-do-not-touch-this§r

§lPlease do not do anything to this village§r.
Thanks!
With love, Feueristic","



    Of course, many were quick to inquire about the village.","§7[Event] Please do not touch this village!§r
§8⛄§r§3Feueristic§r§r §8⌚23/1/26§r
♟§6Ashaz§r Can I touch the village?
♟§8neo§r and you think i'd listen
♟§5Allink§r i think im going to touch this village
♟§akeli5§r i think i will politely look at it
♟§cAlco_Rs11§r that's a lovely screenshot of the village","§7[Event] Please do not touch this village!§r
§8⛄§r§3Feueristic§r§r §8⌚23/1/26§r

♟§8DragonSlayer2189§r
This is an amazing idea, I am simply going to go admire the scenery of this village.","



    At first, the village was only touched in minor ways in game. A §n▁▁▁§r was here sign, perhaps. However, you and I §oboth§r know that this peace was never going to last.","§7[Event] Please do not touch this village!§r
§8⛄§r§3Feueristic§r§r §8⌚23/1/26§r

♟§4lyicx§r §lfuck you fuck the law fuck the policy im anti establishment§r§r
(*thats not my establishment*)

§8§o[Screenshot of a lyicx statue placed in the village]§r§r","


    And, just like that, the floodgates were opened. The statue was placed, T-posed, jarred, etc. Surprisingly, the statue was touched more than the village was. However, the village was still abused in its own way.","


    Farmland borders were replaced with Lyicx-colo§lᵘ§rred concrete. Ores were placed, paths were remade, a cross was placed, lampposts were made, the list goes on.","


      ... however ...","


    it's all... nice. It's a display if the kind of creativity you wish to see come out of a community like this. It's not breathtaking or anything, but it's still positive. The first thought to come to my mind when this event started was: ",'


    "Boy, I wonder how long until this village is griefed and burnt to the ground."

but boy, was I wrong.',"


    While §oPlease do not touch this village§f§0 may have started as a humble forum post, it's blossomed into a small display of what keeps this community as beautiful, yet terrifying as it is.","



      We may have          touched the village,        but the village        touched our hearts        all the same.","§fi used to have this webkinz that i used to pour milk on and suck the milk back out of and it was sopping wet all the time ☹i used to slam it against walls and it would make a loud thud§r
            §lFin§r

§o(Be kind! Rewind!*) 
*this lectern back to page one  "]}
```

**Comment by KadTheHunter** [comment]
Delaying adding this per future decisions around books, see #86

**Comment by KadTheHunter** [comment]
Also should have closed this with d3e1c7712621cdeb5228d51277a4b5ecf2de78b5, closing now.
